CONTRIBUTING.md

```diff
@@ -59,9 +59,9 @@ In this scenario, it’s possible that you found a bug or enhancement that we ha
 ## Working on Expensify Jobs
 *Reminder: For technical guidance please refer to the [README](https://github.com/Expensify/App/blob/main/README.md)*.
 
-#### Express interest for the job on Upwork.com
+#### Check GitHub for existing proposals from other users
 
-1. If you are interested in working on a job posted in Upwork, click **Submit a Proposal** in Upwork to express your interest to the Expensify team.
+1. Expensify reviews all solution proposals on a first come first serve basis. If you see other contributors have already proposed a solution, you can still provide a solution proposal and we will review it. We look for the earliest provided and best proposed solution that addresses the job.
```

Comment:
`earliest provided and best proposed solution that addresses the job`
vs
`earliest provided best proposed solution that addresses the job`

the 1st sounds like focusing in the `earliest`, the 2nd sounds like a bit better IMHO

@iwiznia

Reply:
A comma would probably be good here:

`We look for the earliest provided, best proposed solution that addresses the job.`

Reply:
@jasperhuangg lol self taught English, grammar is not my strong skill 😂

Reply:
Agreed that this seems a bit better ^ here's the suggestion edit so we can commit it quickly :)
```suggestion
1. Expensify reviews all solution proposals on a first come first serve basis. If you see other contributors have already proposed a solution, you can still provide a solution proposal and we will review it. We look for the earliest provided, best proposed solution that addresses the job.
```